packages/firestore/src/local/index_manager.ts

```diff
@@ -22,7 +22,7 @@ import { PersistencePromise } from './persistence_promise';
 /**
  * Represents a set of indexes that are used to execute queries efficiently.
  *
- * Currently the only index is a [collection id] => [parent path] index, used
+ * Currently the only index is a [collection id] =&gt; [parent path] index, used
```

Comment:
Why do we need to escape it? Does it not work in markdown?

Reply:
API Extractor complains about all these HTML-like characters (at least with default settings).